src/fidesops/ops/models/privacy_request.py

```diff
@@ -82,16 +91,16 @@ class ManualAction(BaseSchema):
     update: Optional[Dict[str, Any]]
 
 
-class CollectionActionRequired(BaseSchema):
-    """Describes actions needed on a given collection.
+class CheckpointActionRequired(BaseSchema):
+    """Describes actions needed on a particular checkpoint.
 
     Examples are a paused collection that needs manual input, a failed collection that
     needs to be restarted, or a collection where instructions need to be emailed to a third
     party to complete the request.
     """
 
     step: CurrentStep
-    collection: CollectionAddress
+    collection: Optional[CollectionAddress]
```

Comment:
I'm just using this existing structure that we already have to cache details about a request that paused or failed in the traversal to extend it to describe when a request fails outside of the traversal.  I'm removing the mandate that a collection exists, because this is only relevant inside the traversal.

I've renamed several related pieces.